flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml

```diff
@@ -777,6 +777,7 @@
         <UseTemplate Name="FBW_Push_Toggle">
             <SEQ1_CODE>0</SEQ1_CODE>
             <SEQ2_CODE>(L:XMLVAR_SWITCH_OVHD_INTLT_EMEREXIT_Position) 2 == (L:XMLVAR_SWITCH_OVHD_INTLT_ANNLT_Position) 0 == or</SEQ2_CODE>
+            <TOGGLE_SIMVAR />
```

Comment:
Similarly to Luke's suggestion. It would probably be better to change `FBW_Push_Toggle`, instead of adding these empty `TOGGLE_SIMVAR` elements.

`FBW_Push_Toggle` currently assumes `TOGGLE_SIMVAR` is provided whenever `LEFT_SINGLE_CODE` is not provided. We should probably just move it into its own condition check, instead of the false branch of LEFT_SINGLE_CODE:

```xml
<Condition Check="TOGGLE_SIMVAR">
    <True>
        // ...
    </True>
</Condition>